### DISCUSSION TITLE
KD with pretrained students.

### DISCUSSION THREAD

**Discussion by liguopeng0923** [yoshitomo-matsubara/torchdistill#429] [Q&A] [answered]
Hi @yoshitomo-matsubara ,

I want to know whether there are any distilling methods to fine-tune the pre-trained students, not train them from scratch.

**Comment by yoshitomo-matsubara** [comment]
Hi @liguopeng0923 

For NLP tasks (e.g., GLUE), the following example and paper fine-tune pretrained BERT-Base as a student, using fine-tuned BERT-Large as a teacher

Colab: https://github.com/yoshitomo-matsubara/torchdistill#glue
Paper: https://arxiv.org/abs/2310.17644

**Reply by liguopeng0923** [reply]
Do you know any similar methods for computer vision?

**Reply by yoshitomo-matsubara** [reply, marked as answer]
For instance, the following papers present methods to train modified, pretrained image classification and object detection models as students (only first layers are modified and randomly initialized), learning from the original pretrained teacher models.

https://ieeexplore.ieee.org/abstract/document/9265295/
https://arxiv.org/abs/2007.15818
https://openaccess.thecvf.com/content/WACV2022/html/Matsubara_Supervised_Compression_for_Resource-Constrained_Edge_Computing_Systems_WACV_2022_paper.html

**Reply by liguopeng0923** [reply]
Thank you very much.
